store/wallet.ts

```diff
@@ -22,6 +38,29 @@ export const wallet = createSlice({
   reducers: {
     setStatus: (state, action: PayloadAction<WalletStatus>) => {
       state.status = action.payload
+    },
+    setTokens: (state, action: PayloadAction<AddressToken[]>) => {
+      state.tokens = action.payload
+    },
+    setUtxoBalance: (state, action: PayloadAction<string>) => {
+      state.utxoBalance = action.payload
     }
   }
 })
+
+export const { setStatus, setTokens, setUtxoBalance } = wallet.actions
+
+const rawTokensSelector = createSelector((state: WalletState) => state.tokens, (tokens) => {
+  if (tokens.some((t) => t.id === '0')) {
+    return tokens
+  } else {
+    return [initialDFI, ...tokens]
+  }
+})
+//* Use as token selector to include UTXO balance
+export const tokensSelector = createSelector([rawTokensSelector, (state: WalletState) => state.utxoBalance], (tokens, utxoBalance) => tokens.map((t) => {
```

Comment:
I think we should not merge it, display it separately. My thoughts are people constantly explaining to each other what is UTXO and Account token. We could do that within our design instead, users should not need to reference another help document (telegram) to use the app.

Reply:
This one will be used as a display as the main list of tokens. We can provide a split in display but somewhere we still need to have a total. Also when doing operations, it would still be better to have a total. 

But yeah we can have a breakdown, of current UTXO and Token.

Reply:
We don't need total for any operation as none of the operations allows it. I think the total is more for communicating our design. Still would be useful